sbnanaobj/StandardRecord/SRHeader.h

```diff
@@ -56,6 +57,7 @@ namespace caf
 
       std::string    sourceName; ///< Name of the file or source this event comes from.
       unsigned int   sourceIndex = NoSourceIndex; ///< Index of this event within the source (zero-based).
+      std::uint32_t  sourceNameHash; ///< hash of sourceName, std::hash<std::string>(sourceName), then truncated to std::uint32_t. Should be 32-bit integer to be used as TTreeIndex (https://root.cern/doc/master/classTTreeIndex.html#a08aac749ab22fd5c8ab792a0061a4b0f)
```

Comment:
Could this variable be initialised with something (like 99999) in the constructor list?

Reply:
Thanks for the comments! I set the default as 0 in the new commit.